packages/generators/app/lib/utils/check-requirements.js

```diff
@@ -1,13 +1,15 @@
 'use strict';
 
+const { red, green, bold } = require('chalk');
+
 module.exports = function checkBeforeInstall() {
-  var currentNodeVersion = process.versions.node;
-  var semver = currentNodeVersion.split('.');
-  var major = semver[0];
+  const currentNodeVersion = process.versions.node;
+  const semver = currentNodeVersion.split('.');
+  const major = semver[0];
 
-  if (major < 12) {
-    console.error(`You are running Node ${currentNodeVersion}`);
-    console.error('Strapi requires Node 12 and higher.');
+  if (major < 14 || major > 16) {
```

Comment:
We support node 14 starting from 14.19.1, maybe we should add a check for that?

For reference: `>=14.19.1 <=16.x.x`

Reply:
Oops, didn't saw your comment there, sorry
https://github.com/strapi/strapi/pull/13962#issuecomment-1203873811